docs/content/en/docs/reference/packagespec/harbor/v2.5.1.md

```diff
@@ -0,0 +1,59 @@
+| **General**                                                          |                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |                                       |
+| `externalURL`                                                        | The external URL for Harbor core service                                                                                                                                                                                                                                                                                                                                                                                                                          | `https://127.0.0.1:30003`          |
+| `imagePullPolicy`                                                    | The image pull policy  | `IfNotPresent`                                                                                                                                                                                                                                                                                                                                                                                                                                            |                                       |
+| `logLevel`                                                           | The log level: `debug`, `info`, `warning`, `error` or `fatal`                                                                                                                                                                                                                                                                                                                                                                                                     | `info`                                |
+| `harborAdminPassword`                                                | The initial password of Harbor admin. Change it from portal after launching Harbor                                                                                                                                                                                                                                                                                                                                                                                | `Harbor12345`                         |
```

Comment:
```suggestion
| `harborAdminPassword`                                                | The initial password of the Harbor `admin` account. Change it from the portal after launching Harbor                                                                                                                                                                                                                                                                                                                                                                                | `Harbor12345`                         |
```
If the account name is literally `admin`, I'd suggest the change above. Otherwise, you can take the single quotes off.